widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -70,9 +88,38 @@ export function CustomColorsSection(props: CustomColorsTypes) {
     setOpenCustomColor(null);
   }, [tab]);
 
+  useEffect(() => {
+    setSelectedColors({
+      tab,
+      colors: {
+        light: theme?.colors?.light || {},
+        dark: theme?.colors?.dark || {},
+      },
+    });
+  }, [theme?.colors]);
+
+  const onChangeColor = (
+    name: WidgetColorsKeys,
+    mode: 'light' | 'dark',
+    color?: string
+  ) => {
+    let resetColors = false;
+    if (selectedColors.tab !== tab) {
+      resetColors = true;
```

Comment:
`resetColors = selectedColors.tab !== tab `is not better?

Reply:
yes, is better